torch/onnx/symbolic_helper.py

```diff
@@ -1129,13 +1129,17 @@ def _batchnorm_helper(g, input, weight, bias, running_mean, running_var):
     return weight, bias, running_mean, running_var
 
 
-def _avgpool_helper(tuple_fn, padding, kernel_size, stride, divisor_override, name):
+def _avgpool_helper(
+    tuple_fn: Callable[[Any], Sequence[int]],
```

Comment:
nit: would it be better to be consistent regarding `Sequence` vs `Tuple`?

Reply:
Changed to Sequence